pkg/server/server.go

```diff
@@ -111,6 +111,7 @@ type jsGlobals struct {
 	CustomProductName               string                     `json:"customProductName"`
 	DevCatalogCategories            string                     `json:"developerCatalogCategories"`
 	DevCatalogTypes                 string                     `json:"developerCatalogTypes"`
+	DevConsoleProxyAvailable        bool                       `json:"devConsoleProxyAvailable"`
```

Comment:
We'd want to set a normal console flag rather than server flags since server flags are not an API.

Reply:
You mean a flag using `console.flag/hookProvider` extension?